release_bot/utils.py

```diff
@@ -252,3 +252,16 @@ def update_version(file, new_version):
             output.write('\n'.join(content) + '\n')
         configuration.logger.info('Version replaced.')
     return changed
+
+#parse setup.cfg and return name
+def parse_metadata(file):
+    """
+    Getting Metadata
+    :param file: file is the setup_cfg where the metadata is stored
+    :return: the name from metadata
+    """
+    with open(file, 'r') as setup_cfg:
+        for line in setup_cfg:
+            if "name" in line:
```

Comment:
I'd suggest using [configparser](https://docs.python.org/3/library/configparser.html) for the `setup.cfg`.

Reply:
i should take only the name or i should continue further and take all the metadata to a list?